NetworkRequest/NetworkRequest/Controller/IssueViewController.swift

```diff
@@ -9,12 +9,13 @@ import UIKit
 
 class IssueViewController: UIViewController, UISearchBarDelegate {
     
-    var buttonMore = UIButton()
     var searchText: String = ""
     var page = 1
+    var isLoading = true
```

Comment:
[Question] Why do you need this variable here and in other controllers?

Reply:
fixed